lib/src/main/java/com/cloudbees/groovy/cps/impl/DoWhileBlock.java

```diff
@@ -39,7 +39,7 @@ public Next top() {
             return then(body,e,loopHead);
         }
 
-        public Next loopHead(Object _) {
+        public Next loopHead(Object unused) {
```

Comment:
Forward compatibility for when we eventually compile this with Java 11, as otherwise we get:

```
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/ThrowBlock.java:[38,47] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/SequenceBlock.java:[37,35] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/ValueBoundContinuation.java:[20,31] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/DoWhileBlock.java:[42,36] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/WhileBlock.java:[37,36] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/ForLoopBlock.java:[39,36] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/ForLoopBlock.java:[55,37] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/ForInLoopBlock.java:[60,37] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/ExcrementOperatorBlock.java:[91,32] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/impl/SwitchBlock.java:[101,32] error: as of release 9, '_' is a keyword, and may not be used as an identifier
[ERROR] /home/basil/src/cloudbees/groovy-cps/lib/src/main/java/com/cloudbees/groovy/cps/Next.java:[144,31] error: as of release 9, '_' is a keyword, and may not be used as an identifier
```

Note that compiling with `release=11` yields test failures, as I explained in another comment, so we are still compiling this with Java 8, but this gets us closer to the desired end goal.